howtos/how-to-make-a-clickable-avatar-in-a-header/views/profile.ejs

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>
+        <%= user.name %> | Profile
+    </title>
+
+    <style>
+        .header {
+            display: flex;
+            flex-direction: row;
+            justify-content: space-evenly;
+            align-items: center;
+        }
+
+        .card {
+            box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+            max-width: 300px;
+            margin: auto;
+            text-align: center;
+            font-family: sans-serif;
+            border-radius: 5px;
+        }
+
+        img {
+            border-radius: 50%;
+            margin: 30px auto 0;
+            box-shadow: 0 0 5px 0px #38383875;
+            width: 80px;
+            height: 80px;
+        }
+
+        .title {
+            color: grey;
+            font-size: 18px;
+        }
+
+        button {
+            border: none;
+            border-radius: 5px;
+            outline: 0;
+            display: inline-block;
+            padding: 8px;
+            color: white;
+            background-color: #000;
+            text-align: center;
+            cursor: pointer;
+            width: 95%;
+            font-size: 18px;
+            margin: 10px auto 15px;
+        }
+
+        a {
+            text-decoration: none;
+            font-size: 22px;
+            color: black;
+        }
+
+        button:hover,
+        a:hover {
+            opacity: 0.7;
+        }
+    </style>
+</head>
+
+<body>
+
+    <%- include('shared'); %>
+
+        <div>
+            <div id="user-profile-container">
+                <div class="header">
+                    <a href="/"> ᐸ Friends</a>
+                    <h2>User profile</h2>
+                </div>
+
+                <div class="card">
+                    <img src="<%= user.photoUrl %>" alt="<%= user.name %>">
+                    <h1>
+                        <%= user.name %>
+                    </h1>
+                    <p class="title">
+                        <%= user.email %>
+                    </p>
+                    <p>ID: <%= user.id %>
+                    </p>
+                    <p>Role: <%= user.role %>
+                    </p>
+                    <p>Welcome msg: <%= user.welcomeMessage %>
+                    </p>
+                    <p><button id="open-chat-btn" data-id="<%= user.id %>" data-name="<%= user.name %>"
```

Comment:
Same again about adding click listener here and not reading attributes off html elements

Reply:
this should be resolved now